unblob/processing.py

```diff
@@ -119,3 +149,49 @@ def calculate_unknown_chunks(
         unknown_chunks.append(unknown_chunk)
 
     return unknown_chunks
+
+
+def calculate_entropy(path: Path, *, draw_plot: bool):
+    """Calculate and log shannon entropy divided by 8 for the file in 1mB chunks."""
```

Comment:
Why `divided by 8`?  For bit -> byte conversion?
Since we have this docstring here, could you elaborate it more?

Reply:
Some context which could be used to rephrase the doc-string:

> Shannon entropy returns the amount of information (in bits) of some numeric sequence. We calculate the average entropy of byte chunks, which in theory can contain 0-8bits of entropy. We normalize it for visualization to a 0-100% scale.

Reply:
Yes, shannon entropy outputs the possibility of a value based on the input. So if there are 8 possible values, it will return a number between 1 and 8. When we divide this by 8, we get a "percentage" value, which is easier to interpret IMO. I put something more elaborate in the docstring.